docs/MAASUI.md

```diff
@@ -0,0 +1,177 @@
+# MAAS UI
+
+- Content
```

Comment:
You can probably remove "content" as a bullet point here and just change it to a L2 heading:

```markdown
## Contents
- [Usability](#usability)
- [Code structure](#code-structure)
- etc...
```

Reply:
Sure, thanks.. I'll work on that